sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -304,6 +304,27 @@ Licensed under the MIT License.
         </plugins>
       </build>
     </profile>
+    <profile>
```

Comment:
for personal understanding: do we still need these in `azure-cosmos`? because all tests are in `azure-cosmos-tests `now

Reply:
Actually, we don't, let me clean up with this effort, so we don't have to come back to this, thanks @xinlian12

Reply:
Done, removed all the unused resources as well.